data/xml/2023.bea.xml

```diff
@@ -590,7 +590,7 @@
       <title>Learning from Partially Annotated Data: Example-aware Creation of Gap-filling Exercises for Language Learning</title>
       <author><first>Semere Kiros</first><last>Bitew</last><affiliation>Ghent University - imec, IDLab</affiliation></author>
       <author><first>Johannes</first><last>Deleu</last><affiliation>Ghent University - imec</affiliation></author>
-      <author><first>A. Seza</first><last>Doruz</last><affiliation>Universiteit Gent</affiliation></author>
+      <author><first>A.Seza Doğruöz</first><last>Do</last><affiliation>Universiteit Gent</affiliation></author>
```

Comment:
This doesn't seem quite right:
* There should be a space after `A.` in the first name part
* The last name should be within the last name field